data/pc/common/features.json

```diff
@@ -333,5 +333,31 @@
     "name": "sendStringifiedSignText",
     "description": "sign text send when updating signs is send as stringified strings",
     "versions": ["1.8", "1.8.9"]
+  },
+  {
+    "name": "metadataIxOfItem",
+    "description": "item.metadata[this_ix] will be the item that was dropped on the ground",
+    "values": [
+      {
+        "value": 8,
+        "versions": ["1.17", "latest"]
+      },
+      {
+        "value": 7,
+        "versions": ["1.14", "1.16"]
```

Comment:
are you sure you don't mean like 1.14_major to 1.16_major ?

Reply:
yep, fixed